Directory.Build.props

```diff
@@ -34,7 +34,7 @@
     <!-- Default to all packages generating a corresponding symbol package -->
     <IncludeSymbols>true</IncludeSymbols>
     <IsShippingPackage>false</IsShippingPackage>
-    <SdkTargetFramework>net9.0</SdkTargetFramework>
+    <SdkTargetFramework>net10.0</SdkTargetFramework>
```

Comment:
I think you also want to set `NetCurrent` here to `net10.0` as at least one project (dotnet-format) uses that property.